joblib/_utils.py

```diff
@@ -0,0 +1,37 @@
+# Adapted from https://stackoverflow.com/a/9558001/2536294
+
+import ast
+import operator as op
+
+# supported operators
+operators = {
+    ast.Add: op.add,
+    ast.Sub: op.sub,
+    ast.Mult: op.mul,
+    ast.Div: op.truediv,
```

Comment:
Maybe also allow Euclidean div.

Reply:
not sure what euclidean div is, but added floordiv and mod.

Reply:
Those are what I meant.